### PR TITLE
Issue #97: Full refactor of person to candidate across files

### LINKED ISSUE #97: Further refactor: `Person` -> `Candidate`
To look deeper into this.

For further discussion with team: Is it necessary?

E.g. There is a file `src/main/java/seedu/address/storage/JsonAdaptedPerson.java`.

### PR DESCRIPTION
**Pull request by charmainehly**
This PR addresses the full refactor `person` to `candidate` across all files.

**Excluded directories:** .`gradle/`and `build/`

**Case sensitive search and replace:** `Person`, `person`, `PERSON`

Closes #97 

_Note to team: many line changes although there should be no impact on functionality. Would be good to get everyone's help to make sure the folders - especially `docs/` and `src/` have no errors :) Thanks!!_

### CODE REVIEW

**Comment by charmainehly** [comment]
Just an additional note, given as it's probably quite difficult to manually check through, perhaps what would be good is if we can all run through the documentation files and run the function to make sure everything is functioning/displaying as expected! :)

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -447,7 +447,7 @@ This feature has been enhanced based on the initial implementation of the `help`
 #### How is this feature implemented?
 
 The proposed `help` mechanism is facilitated by `HelpWindow`. It is supported by `HelpWindowUtil` where it will help the user open the User Guide on their desktop. 
-We used the existing implementation, and we modified it that the user can view the full user guide on their personal desktop with the click of a button. 
+We used the existing implementation, and we modified it that the user can view the full user guide on their candidateal desktop with the click of a button.
```
Comment (LeeEnEn) [inline docs/DeveloperGuide.md:450]: This might be a typo. 🤔

**Review by LeeEnEn — CHANGES_REQUESTED** [review]
Thank you for hard work on the major refactor!
Only managed to spot a small spelling mistake. 🥂

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (charmainehly) [reply docs/DeveloperGuide.md]: Thanks @LeeEnEn  :) It must have been during the replacement from `person` --> `candidate`, but missed out on checking for `personal`! Have edited this!

**Review by LeeEnEn — APPROVED** [review]
LGTM! 👍

**Review by lzan98 — APPROVED** [review]
Thank you for the hard work!! 😄 I downloaded your branch to test the app locally and it works fine so there shouldn't be any issues! (hopefully)

**Pull request closed without merging**